packages/camera/android/src/main/java/io/flutter/plugins/camera/Camera.java

```diff
@@ -392,6 +392,34 @@ public void stopVideoRecording(@NonNull final Result result) {
     }
   }
 
+  public void pauseVideoRecording(@NonNull final Result result) {
+    if (!recordingVideo) {
+      result.success(null);
+      return;
+    }
+
+    try {
+      mediaRecorder.pause();
+      result.success(null);
```

Comment:
nit: This should be outside of the `try` block here and below, just in the weird offhand chance calling this throws an `IllegalStateException`. If it did `result.error` would also throw in the `catch` block.